.github/workflows/storybook-tests.yml

```diff
@@ -0,0 +1,20 @@
+name: 'Storybook Tests'
+on: push
+jobs:
+  test:
+    timeout-minutes: 60
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - uses: actions/setup-node@v3
+      - name: Install dependencies
+        run: yarn
+      - name: Install Playwright
+        run: npx playwright install --with-deps
+      - name: Build Storybook
+        run: yarn build-storybook --quiet
+      - name: Serve Storybook and run tests
+        run: |
+          npx --yes concurrently -k -s first -n "SB,TEST" -c "magenta,blue" \
+            "npx --yes http-server storybook-static --port 6006 --silent" \
+            "npx --yes wait-on http://127.0.0.1:6006 && yarn test-storybook"
```

Comment:
These are failing on CI - I imagine it's some snapshot config needs sorting out.

Could you also add an `@actions/upload-artifact` action for the output of the playwright run? I assume it'll output some sort of diff photos or other useful test output that we could store as an output for a few days.